### PR TITLE
Remove Unnecessary Dependencies

### PR DESCRIPTION
**Pull request by harry1453**

### CODE REVIEW

File: burst.sh [hunk 1]
```diff
@@ -419,10 +419,6 @@ if [[ $# -gt 0 ]] ; then
             ;;
     esac
 else
-    ARCH=`uname -m`
-    if [[ $ARCH = "armv7l" ]]; then
-        export LD_LIBRARY_PATH=./lib/armv7l
-    fi
```
Comment (blankey1337) [inline burst.sh]: not needed?

Comment (harry1453) [reply burst.sh]: The variable is not referenced anywhere else, so I think not.

File: src/brs/util/StringUtils.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
+package brs.util;
+
+public class StringUtils {
```
Comment (ohager) [inline src/brs/util/StringUtils.java]: A very good candidate for a unit test ;-) (just to bother you)

Comment (harry1453) [reply src/brs/util/StringUtils.java]: I'll need to do a ton of unit tests soon anyway... ;)

**Pull request merged**